samples/java_layout/src/main/java/io/customer/android/sample/java_layout/data/model/User.java

```diff
@@ -0,0 +1,57 @@
+package io.customer.android.sample.java_layout.data.model;
+
+import android.text.TextUtils;
+
+import androidx.annotation.Nullable;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class User {
+    private static class Keys {
```

Comment:
we can probably, remove the separate class and have constats?

Reply:
It actually is not required by any other class than `User`. So I would suggest not to move it outside the `User` class. I can replace it with static constants if you like?